### PR TITLE
update TORA model w/simulation results

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Review by schillic — APPROVED** [review]

**Review by schillic — APPROVED** [review]

File: models/Sherlock-Benchmark-9-TORA/Sherlock-Benchmark-9-TORA.jl [hunk 1]
```diff
@@ -29,13 +29,14 @@
...
-    dx[3] = x[4]
+    x₁, x₂, x₃, x₄, u = x
+
+    aux = 0.1 * sin(x₃)
+    dx[1] = x₂
+    dx[2] = -x₁ + aux
+    dx[3] = x₄
     dx[4] = u
-    return dx
+    dx[5] = zero(u)
```
Comment (schillic) [inline models/Sherlock-Benchmark-9-TORA/Sherlock-Benchmark-9-TORA.jl:39]: Do you need to return `dx`?

File: src/setops.jl [hunk 2]
```diff
@@ -2,13 +2,13 @@
 # Projection operations
 # ========================================
 
-function _Projection(X::AbstractLazyReachSet, st_vars)
-    Projection(X, st_vars)
+function _project_oa(X::AbstractLazyReachSet, vars)
+    return Project(X, vars)
```
Comment (schillic) [inline src/setops.jl:6]:
```suggestion
    return project(X, vars)
```
Not sure if you wanted to have a concrete set here. At least you removed the comment "lazy set".

**Review by schillic — APPROVED** [review]

File: models/Sherlock-Benchmark-9-TORA/Sherlock-Benchmark-9-TORA.jl (hunk 1 above)
Comment (mforets) [reply models/Sherlock-Benchmark-9-TORA/Sherlock-Benchmark-9-TORA.jl:39]: no, it doesn't need to as it modifies dx inplace

File: src/setops.jl (hunk 2 above)
Comment (mforets) [reply src/setops.jl:6]: i think we can keep the lazy set here. and let the network decide how to handle it

**Pull request merged**